### PR TITLE
Bootstrap icons  v1.0.0-alpha2

### PR DESCRIPTION
**Pull request by meetdilip**
Added suggested updates to the SVG file. Please review.

### CODE REVIEW

File: Bootstrap/Bootstrap-OpenSymbols.svg [hunk 1]
```diff
@@ -0,0 +1,4038 @@
...
+       id="path9" />
+    <path
+       fill-rule="evenodd"
+       d="M13.646 16.146a.5.5 0 01.708 0l1 1a.5.5 0 01-.708.708l-1-1a.5.5 0 010-.708zm-7.292 0a.5.5 0 00-.708 0l-1 1a.5.5 0 00.708.708l1-1a.5.5 0 000-.708zM7.5 2.5A.5.5 0 018 2h4a.5.5 0 010 1H8a.5.5 0 01-.5-.5z"
+       clip-rule="evenodd"
+       id="path11" />
+    <path
+       d="M9 3h2v2H9V3z"
+       id="path13" />
+  </symbol>
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-OpenSymbols.svg]: Did you test it? Seems like the set is not well formed.  This symbol is wrongly closed in line 63

File: Bootstrap/Bootstrap-OpenSymbols.svg [hunk 2]
```diff
@@ -0,0 +1,4038 @@
...
+  <symbol
+     id="alert-circle-fill"
+     viewBox="0 0 20 20">
+    <path
+       fill-rule="evenodd"
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-OpenSymbols.svg]: You did not remove this `fill-rule="evenodd"`

File: Bootstrap/Bootstrap-OpenSymbols.svg [hunk 3]
```diff
@@ -0,0 +1,4038 @@
...
+       d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-8.998 3a1 1 0 112 0 1 1 0 01-2 0zM10 6a.905.905 0 00-.9.995l.35 3.507a.553.553 0 001.1 0l.35-3.507A.905.905 0 0010 6z"
+       clip-rule="evenodd"
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-OpenSymbols.svg]: You did not remove this `clip-rule="evenodd"`

File: Bootstrap/Bootstrap-OpenSymbols.svg [hunk 4]
```diff
@@ -0,0 +1,4038 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg1479"
+   sodipodi:docname="Bootstrap-icons.svg"
+   inkscape:version="0.92.4 (unknown)">
+  <title
+     id="title102576">Bootstrap Icons</title>
+  <metadata
+     id="metadata1485">
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-OpenSymbols.svg]: You did not remove this id

File: Bootstrap/Bootstrap-OpenSymbols.svg [hunk 5]
```diff
@@ -0,0 +1,4038 @@
...
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg1479"
+   sodipodi:docname="Bootstrap-icons.svg"
+   inkscape:version="0.92.4 (unknown)">
+  <title
+     id="title102576">Bootstrap Icons</title>
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-OpenSymbols.svg]: Please remove this `id`

**Comment by meetdilip** [comment]
If you don't mind, can you highlight the exact portion which needs to be removed? I was removing from < tag to tag>. Perhaps, that messed up the file.

**Comment by meetdilip** [comment]
I used an online SVG cleaning tool. Attaching the file. Please review. Possible that unwanted code is gone

[Bootstrap-icons.zip](https://github.com/Xaviju/inkscape-open-symbols/files/4159218/Bootstrap-icons.zip)

**Comment by meetdilip** [comment]
Shall I close this PR ?

**Comment by meetdilip** [comment]
How should we proceed ?

**Pull request merged**